lib/src/arithmetic/carry_save_mutiplier.dart

```diff
@@ -110,7 +108,7 @@ class CarrySaveMultiplier extends Multiplier {
       _pipeline.get(rCarryB),
     );
 
-    product <=
+    output('product') <=
```

Comment:
you can reference `product` directly instead of calling `output('product')`, right?

Reply:
I do run into some challenges accessing the getter in a subclass, so I use the output('').  But it looks like I have found a way to eliminate these, though I think I am leveraging some aliases I need to clean up (e.g. fp_mult:  internalProduct.)

Reply:
I found the problem I was facing:   the fp_simple has a getter that converts from output('sum') to a floating point.
So we can't use the getter in setting operations.